.github/workflows/commit-check.yaml

```
@@ -0,0 +1,87 @@
+        run: |
+          branch_name="${{ env.branch_name }}"
+          issue_number=$(echo "$branch_name" | cut -d '/' -f 2)
+          PR_NUMBER=${{ github.event.pull_request.number }}
+          PR_AUTHOR=${{ github.event.pull_request.user.login }}
+
+          # Allowed commit title keywords
+          ALLOWED_KEYWORDS="^(feat|fix|refactor|chore|docs|test|style|perf|revert)\\(.+\\): .{10,}$"
+          INVALID_COMMITS=0
+          MISSING_ISSUE_NUMBER=0
+          COMMENT_BODY=""
+          
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)